src/posposet/common_test.go

```diff
@@ -0,0 +1,308 @@
+package posposet
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/Fantom-foundation/go-lachesis/src/common"
+)
+
+func TestParseEvents(t *testing.T) {
+	assert := assert.New(t)
+
+	nodes, events, names := ParseEvents(`
```

Comment:
Hmm, well that's certainly more condensed. I suppose that's readable and succinct enough to include in docstrings without replacing with a URL. Neat!